### PR TITLE
bump browser versions

### PR DESCRIPTION
**Pull request by digitronik**
@quarckster I need chrome only but even I touch firefox
please correct me

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -10,15 +10,19 @@ jobs:
...
     strategy:
       matrix:
-        firefox_version: ["68.12.0esr", "84.0.2", "78.6.1esr"]
-        chrome_version: ["88.0.4324.96"]
-        gecko_version: ["v0.28.0", "v0.29.0"]
+        # Firefox releases
+        # https://download-installer.cdn.mozilla.net/pub/firefox/releases/
+        # Chrome versions
+        # https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable
+        firefox_version: ["68.12.0esr", "84.0.2", "78.9.0esr", "87.0"]
```
Comment (quarckster) [inline .github/workflows/deploy.yml]: Old versions can be dropped:
```
firefox_version: ["68.12.0esr", "78.9.0esr", "87.0"]
```

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -10,15 +10,19 @@ jobs:
...
+        chrome_version: ["88.0.4324.96", "89.0.4389.128-1"]
```
Comment (quarckster) [inline .github/workflows/deploy.yml]: Same here
```
chrome_version:  ["89.0.4389.128-1"]
```

**Review by quarckster — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (digitronik) [reply .github/workflows/deploy.yml]: sure

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (digitronik) [reply .github/workflows/deploy.yml]: @quarckster done

**Review by quarckster — APPROVED** [review]

**Pull request merged**